README.md

```diff
@@ -0,0 +1,286 @@
+
+# Bridgewell SDK for Android
+
+[Links to the full documentation](https://github.com/bridgewell/Android-SDK/blob/main/README.md)
+
+**Version: 1.0**
+
+<u><b>Note</b></u>: All integration examples are written in Kotlin. If you’re using Java please convert them to corresponding Java code
+
+## Table of Contents
+
+- [Requirements](#requirements)
+- [General Setup](#general-setup)
+    - [1. SDK installation](#1-sdk-installation)
+        - [a. Using Maven Central](#a-using-maven-central)
+        - [b. Using the .aar file](#b-using-the-aar-file)
+    - [2. SDK initialization](#2-sdk-initialization)
+        - [a. Set BW Server](#a-set-bw-server)
+        - [b. Initialize SDK](#b-initialize-sdk)
+        - [c. Geolocation sharing](#c-share-geo-location)
+    - [3. Update Your Android Manifest](#3-update-your-android-manifest)
+- [SDK Usage](#sdk-usage)
+    - [1. WebView API usage](#1-web-view-api)
+    - [2. Banner ads](#2-banner-ads)
+        - [a. Display normal banner](#a-display-normal-banner)
+        - [b. Sticky banner](#b-sticky-banner)
+- [About](#about)
+
+
+<a id="requirements"></a>
+## Requirements
+
+
+- Android 8.0 (SDK version 26) or newer
+- Mobile phone with Google Play Services
+- Android Studio 2023.3.1 or newer
+- Gradle 8.2 or newer
+<br>
+
+<a id="general-setup"></a>
+# General setup
+
+<a id="1-sdk-installation"></a>
+### 1. SDK installation
+<a id="a-using-maven-central"></a>
+#### a. Using Maven Central
+1.  Add the dependency:
+
+```gradle
+dependencies {
+	implementation("com.bridgewell:bwmobile:1.3.0")
+}
+```
+
+2. Sync your project.
+
+
+<a id="b-using-the-aar-file"></a>
+#### b. Using the .aar file
+
+1. Place the `.aar` file into your project's `app/libs` folder: `/app/libs/bwmobile.aar`
+2. Add this code to the dependencies block in your application's `build.gradle` file:
+
+```gradle
+dependencies {
+    implementation fileTree(dir: "libs", include: ["*.jar", "*.aar"])
+    implementation files('libs/bwmobile.aar')
+}
+```
+
+3. Sync your project.
+
+
+<a id="2-sdk-initialization"></a>
+### 2. SDK initialization
+
+<a id="a-set-bw-server"></a>
+#### a. Set the BW Server
```

Comment:
We have already specified the BW server within the SDK, and we prefer that users do not make any changes to it